docs/reference_guide.md

```diff
@@ -28,7 +28,7 @@ discussion to other files in /docs, the /tools/\*\_examples.txt files, or blog p
     - [2. `/.../`: Filtering](#2--filtering)
     - [3. `//`, `/*`: Comments](#3---comments)
     - [4. Literals](#4-literals)
-    - [5. `->`: C Struct Navigation](#5---c-struct-navigation)
+    - [5. `.`: C Struct Navigation](#5---c-struct-navigation)
```

Comment:
The `->` operator is still available and has the same semantics as in C (dereference + field access). The only recent change is that `.` is now the default way of accessing fields of builtin `args` (while `->` works too, for backwards compatibility).
```suggestion
    - [5. `.`, `-.>`: C Struct Navigation](#5---c-struct-navigation)
```